tests/integration/nm/vlan_test.py

```diff
@@ -35,7 +35,7 @@ def test_create_and_remove_vlan():
         assert vlan_desired_state == vlan_current_state
 
     # FIXME: Remove this sleep after adding wait for state mechanism.
-    time.sleep(2)
+    time.sleep(5)
```

Comment:
This should probably be a separate PR/commit, since it is unrelated/fixes problems for other PRs as well.

Reply:
Right